### PR TITLE
Add height/width attributes to HTML source element

### PR DESCRIPTION
**Pull request by dletorey**
#### Summary

Added browser compat data for height & width attribute on 

#### Test results and supporting details

used `npm run test` to run unittests locally 

- [Bugzilla 1694741](https://bugzilla.mozilla.org/show_bug.cgi?id=1694741)
- [Content Issue #20927](https://github.com/mdn/content/issues/20927)

### CODE REVIEW

**Review by queengooborg — CHANGES_REQUESTED** [review]
Hey there, thank you for your PR and welcome to BCD!  I was able to confirm Chrome support based on https://chromestatus.com/feature/5737185317748736 and Firefox support from the linked bug.  Could you provide sources for where "Safari 15" had come from?

Also, we can mirror Opera Android as well:

**Comment by dletorey** [comment]
Ah yes, I confirmed Safari by testing in Browserstack, I am not positive about Opera Android, I shall test and confirm when working next.
Thank you

File: html/elements/source.json [hunk 1]
```diff
@@ -37,6 +37,47 @@
...
+              "firefox": {
+                "version_added": "106",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "dom.picture_source_dimension_attributes.enabled",
+                    "value_to_set": "true"
+                  }
+                ]
+              },
```
Comment (hamishwillee) [inline html/elements/source.json:59]: HI Dave, 
We now know this is shipping in 108 too so I'm going to merge updates into your PR.

```suggestion
              "firefox": [
                {
                  "version_added": "108"
                },
                {
                  "version_added": "106",
                  "version_removed": "108",
                  "flags": [
                    {
                      "type": "preference",
                      "name": "dom.picture_source_dimension_attributes.enabled",
                      "value_to_set": "true"
                    }
                  ]
                }
              ],
```

File: html/elements/source.json [hunk 2]
```diff
@@ -226,6 +267,49 @@
...
```
Comment (hamishwillee) [inline html/elements/source.json:290]:
```suggestion
              "firefox": [
                {
                  "version_added": "108"
                },
                {
                  "version_added": "106",
                  "version_removed": "108",
                  "flags": [
                    {
                      "type": "preference",
                      "name": "dom.picture_source_dimension_attributes.enabled",
                      "value_to_set": "true"
                    }
                  ]
                }
              ],
```

**Comment by hamishwillee** [comment]
@dletorey I added the FF108 BCD stuff as suggestions - can you accept those please?

When are the remaining changes to get this in planned?

**Comment by bsmth** [comment]
It looks like I don't have permissions to merge the suggestions, so this can wait for @dletorey to apply.

Also @queengooborg should we be removing the flag data here per [flag guidelines](https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines/index.md#removal-of-irrelevant-flag-data)?

>once a feature has landed in a stable browser release, the flag data quickly becomes irrelevant and may be removed from BCD

**Comment by queengooborg** [comment]
Yes, we should remove the flag data per the guidelines to help consolidate the data.  Firefox 108 is only one release away!

File: html/elements/source.json (hunk 1 above)
Comment (dletorey) [reply html/elements/source.json:59]: @hamishwillee thanks for this and it also makes things much clearer too.

**Review by bsmth — APPROVED** [review]
Looks good thanks 👍🏻

File: html/elements/source.json [hunk 3]
```diff
@@ -226,6 +269,49 @@
...
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": "9"
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": {
+                "version_added": null
+              },
```
[the remark this answers is not in the document]
Comment (queengooborg) [reply html/elements/source.json:285]: Opera Android is simply a Chrome Android fork, so something as basic as this should most certainly be enabled.  (It wouldn't make much sense for them to disable it.)  Thus, it's totally safe to mirror!

**Comment by hamishwillee** [comment]
> Yes, we should remove the flag data per the guidelines to help consolidate the data. Firefox 108 is only one release away!

@queengooborg IMO what we should always add the flag data and use tooling to sweep through and remove the prefs when releases are made. Why? 
- You're always going to have some omissions so you need to tool anyway. 
- If this went in now, there would be 1 month where there would be no information on MDN about how to enable the preference and the release build doesn't have it. So we're basically "wrong" for a month.
- Its easier to revert if the feature doesn't go into a release, because the information is right there. These things get reverted sometimes so it is not a theoretical problem.

File: html/elements/source.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (hamishwillee) [reply html/elements/source.json:285]: @bsmth This ^^^^

[the remark this answers is not in the document]
Comment (hamishwillee) [reply html/elements/source.json:285]: @dletorey Sorry, I mean "can you merge suggestion"? 

FYI, normally I can merge suggestions, and it is nice to be able to do so when you know the fix is good. Saves the round tripping. Must be something restrictive about your fork

[the remark this answers is not in the document]
Comment (bsmth) [reply html/elements/source.json:285]: Is it this setting "Allow edits by maintainers"? I see this on the right hand side of a pull request under Reviewers, Assignees etc.
image

[the remark this answers is not in the document]
Comment (hamishwillee) [reply html/elements/source.json:285]: I think so.

[the remark this answers is not in the document]
Comment (bsmth) [reply html/elements/source.json:285]: I landed on [the GitHub docs page for this setting](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests) by chance yesterday so I'm adding for posterity.

[the remark this answers is not in the document]
Comment (queengooborg) [reply html/elements/source.json:285]: That checkbox _should_ allow us to apply suggestions and edits, but if I had to guess, perhaps it's something to do with the fact that the repository is forked to an organization (@CodeRedDigital) that has more restrictive permissions, so you as the author will have to apply suggestions and edits yourself, it seems?

**Review by queengooborg — APPROVED** [review]
LGTM, thank you!

**Pull request merged**